code/src/java/plugin/dicebag/gui/DiceBagModel.java

```diff
@@ -233,10 +215,9 @@ public double rollDie(int index)
 	 * @param expression Die expression to parse
 	 * @return The double value of the expression.
 	 */
-	public double rollDie(String expression)
+	static double rollDie(String expression)
```

Comment:
This is silly.  If this is going to be converted to static, it should just directly call the static in RollingMethods - in other words, this static method should be inlined.